UKHomeOffice/sgmr-service
# COP-2763: Change vessel nationality text field to a select

## Description

Replace the text field with a select element for vessel nationality field.

## Testing

1. Go to the new report form
2. Go to the vessel step
3. Check if the vessel nationality field is a select element

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
